CHANGELOG.md

```diff
@@ -1,10 +1,29 @@
 Changelog
 =========
 
+## 6.1.3 (2020-08-17)
+
+### Bug fixes
+
+* Persist user when set on client
```

Comment:
I believe these changes only affect React Native? If so, we should prefix them (e.g. "(React Native) Persist user when set on client") to make it clear the issue doesn't affect vanilla Cocoa.

Reply:
That's not the case for this particular change but I've updated the ones which are RN specific as requested.